test/isolation/specs/deadlock_dropchunks_select.spec

```diff
@@ -16,12 +16,12 @@ teardown
 { DROP TABLE DT; DROP table ST; DROP table SL; }
 
 session "s1"
-setup	{ BEGIN; SET TRANSACTION ISOLATION LEVEL READ COMMITTED; SET LOCAL lock_timeout = '50ms'; SET LOCAL deadlock_timeout = '10ms'; }
+setup	{ BEGIN; SET TRANSACTION ISOLATION LEVEL READ COMMITTED; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms'; }
```

Comment:
FWIW, this timeout seems like something we'd like to set globally, per test, or  maybe even in postgresql.conf. 

Not strictly necessary to fix in this PR, but if you're touching all these files, why not just make it a variable or global default instead?

Reply:
this is specific to isolation tests so it certainly cannot be a global also since the isolation tests dont use psql they dont have variable support so that would require changing all the tests to template files and let cmake generate them, so definitely a task for a separate PR